releases/net471/KnownIssues/470006-GC Crashes with high rate of large object allocation.md

```diff
@@ -0,0 +1,36 @@
+# Rare crashes can occur when background GC is enabled in applications which allocate and free large objects at a high rate
+
+## Symptoms
+
+When background GC is enabled, an application running against .NET Framework 4.7.1 experiences
+intermittent crashes which did not occur when running against other .NET Framework versions.
+
+## Cause
+
+Changes to the GC in .NET Framework 4.7.1 caused this problem.
```

Comment:
There's no need for this paragraph. This section can begin, ".NET Framework 4.7.1 includes changes that improve background GC performance. Because of these changes, a rare combination of factors can cause the background GC..."